pkg/helper/boolean_utils.go

```diff
@@ -0,0 +1,17 @@
+package helper
+
+func Any(arr []bool) bool {
+	result := false
+	for _, v := range arr {
+		result = result || v
```

Comment:
Is this intended to check whether at least one of the bools in the array is `true`? if so, we can just check it and return without needing to iterate all the array.
Same would happen with `All` but just checking the opposite.

Reply:
right, this is an alternative implementation. But both approaches work, right?

Reply:
both should work yes. I was pointing there was no need to iterate all array